content/Examples/python/index.mdx

```diff
@@ -0,0 +1,219 @@
+---
+title: Python SDK Examples
+description: Examples for using the Agentuity Python SDK
+---
+
+# Python SDK Examples
+
+This section contains examples for using the Agentuity Python SDK.
+
+## Basic Agent
+
+A simple agent that returns a greeting.
+
+```python
+from agentuity import AgentRequest, AgentResponse, AgentContext
+
+async def run(request: AgentRequest, response: AgentResponse, context: AgentContext):
+    # Get the request data
+    data = request.data.json
+    name = data.get("name", "Guest")
+    
+    # Log the request
+    context.logger.info(f"Received greeting request for {name}")
+    
+    # Return a personalized greeting
+    return response.json({
+        "message": f"Hello, {name}! Welcome to Agentuity.",
+        "timestamp": context.now().isoformat()
```

Comment:
context doesn't have a now function. maybe use datetime.now().isoformat()? make sure you update imports